DigitalLearningSolutions.Data/Helpers/LearningHubSsoSecurityHelper.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace DigitalLearningSolutions.Data.Helpers
+{
+    using System;
+    using System.Security.Cryptography;
+    using System.Text;
+    using DigitalLearningSolutions.Data.Services;
+    using Microsoft.Extensions.Configuration;
+
+    public interface ILearningHubSsoSecurityHelper

```

Comment:
Ah, okay, so this escaped me when I made the ticket.

This doesn't fit the Helper pattern we've been using - they're all static classes.
This is really a service, in that you're registering it with DI and are using instances of it.

Can you move it to the services folder and tweak the name accordingly?